src/Configuration/DefaultConfiguration.php

```diff
@@ -272,6 +272,10 @@ public function controllersToRemove(array $paths) : self
 
         $localRelativePaths = array_map(function ($absolutePath) {
             $relativePath = str_replace($this->localProjectDir, '', $absolutePath);
```

Comment:
Hi, i wonder why don't we just replace this line to `$relativePath = mb_substr($absolutePath, mb_strlen($this->localProjectDir));` instead?

Reply:
Ok i get it now. Your solution indeed fix the issue you mentioned.